app/components/ProgressBars.tsx

```diff
@@ -1,3 +1,24 @@
-// This is where you will implement the progress bars (there are 5 total)
+const ProgressBar = ({ progress }) => {
+  const container = {
+    height: '10px',
+    width: '80%',
+    backgroundColor: '#e0e0df',
+    overflow: 'hidden',
+  };
 
-// Write your code here
\ No newline at end of file
+  const fill = {
+    height: '100%',
+    width: `${progress}%`,
+    backgroundColor:'#65a30cff' ,
+  };
+
+  return (
+    <div style={container}>
+      <div style={fill}>
+      </div>
+    </div>
+  );
```

Comment:
@alexanderviolet @zoyaamrit Hi guys! Can you please make this Tailwind structure instead of using `style`? Instead of creating constants for `container` and `fill`, for our project, we're using the style convention of:

`<div className="...(insert styling here)...">
      <div
        className="...(insert styling here)..."
        style={{ width: `${progress}%` }}
      ></div>
    </div>`